guide/src/python_from_rust.md

```diff
@@ -281,7 +281,7 @@ The example below shows:
   imported from `utils/foo.py`
 
 `src/main.rs`:
-```ignore
+```rust
```

Comment:
I suspect these were intended to be ignored. I am not sure about the correct syntax, but maybe

~~~
```rust,ignore
...
```
~~~

which achieve the desired behaviour and the desired presentation?

Reply:
(This does indeed appear to be the reason why the CI is failing.)

Reply:
Hey @adamreichold  -- thanks for the feedback.

This `rust,ignore` format appears [valid](https://github.com/rust-lang/crates.io/issues/1150), but I'm not familiar with it.

I'm not seeing this work on my end in rendering(could just be VS Code not rendering the tags?)
![image](https://user-images.githubusercontent.com/523300/189491529-77431b03-7520-49a1-b6b2-ceecbc1c31ae.png)

I updated the PR per your request, please let me know if this is better/working in the github formatter or I should make another change?

Reply:
This does indeed seem to render on [git](https://github.com/PyO3/pyo3/blob/f0ee2d037dc31bc7ccabdc76b931cded0344936e/guide/src/python_from_rust.md)

Reply:
Oops, I messed up the force push. Just going to start fresh with a new PR, my sincere apologies.